### ISSUE TITLE
ImportError: No module named rtmidi

### ISSUE THREAD

**Issue by ToXIc-Dev** [mido/mido#239]
Would have used the google group but it says it doesn't exist or I don't have permission.

pip3 install python-rtmidi returns: Requirement already satisfied: python-rtmidi

**Comment by bast** [comment]
Is the problem still there? If yes, you can also ask on "Discussion".

**Comment by Liujingxiu23** [comment]
Yes, problem still exist, and when run  “pip install python-rtmidi “， it can not be installed successfully
